index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Inter"
+      rel="stylesheet"
+    />
+    <title>Portfolio: setup and mobile first</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a href="#" class="nav-branding inter-font">Welcome</a>
+        <div class="humburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <section class="headline inter-font">
+      <h1>
+        Hey there. I’m <br>Manar
+        <span class="greenTxt"> Iam a software <br>developer</span>
+      </h1>
+      <p class="inter-font">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="socialMedia">
```

Comment:
- [ ] Please improve styling in order to look mire like the design
Yours: 
![image](https://user-images.githubusercontent.com/62705947/184912066-0d1d6a87-1f38-4597-b55c-3ef4a9b3f964.png)
Design:
![image](https://user-images.githubusercontent.com/62705947/184912166-8c0bef6d-4ec0-42dc-afbd-7859eaf4e3e8.png)

---------------------------------------------------
![image](https://user-images.githubusercontent.com/62705947/184912836-32a8af9f-8f59-43d1-9d6e-b54c0c2e138b.png)

![image](https://user-images.githubusercontent.com/62705947/184913422-35bf728a-fb6b-402f-b739-b4b6181ec068.png)

Reply:
fixed